config/kubernetes/org.yaml

```diff
@@ -1705,6 +1706,7 @@ teams:
     members:
```

Comment:
can you please update the group description suggesting that this team is used for releases management.

Reply:
done